etc/storage.conf

```diff
@@ -1,31 +1,137 @@
+# This file is is the configuration file for all tools
+# that use the containers/storage library.
+# See man 5 containers-storage.conf for more information
+# The "container storage" table contains all of the server options.
 [storage]
-  driver = "overlay"
-  runroot = "/var/tmp/theia"
-  graphroot = "/home/theia/.local/share/containers/storage"
-  [storage.options]
-    size = ""
-    remap-uids = ""
-    remap-gids = ""
-    ignore_chown_errors = "true"
-    remap-user = ""
-    remap-group = ""
-    ostree_repo = ""
-    skip_mount_home = ""
-    mount_program = "/usr/bin/fuse-overlayfs"
-    mountopt = ""
-    [storage.options.thinpool]
-      autoextend_percent = ""
-      autoextend_threshold = ""
-      basesize = ""
-      blocksize = ""
-      directlvm_device = ""
-      directlvm_device_force = ""
-      fs = ""
-      log_level = ""
-      min_free_space = ""
-      mkfsarg = ""
-      mountopt = ""
-      use_deferred_deletion = ""
-      use_deferred_removal = ""
-      xfs_nospace_max_retries = ""
 
+# Default Storage Driver
+driver = "vfs"
+
+# Temporary storage location
+# runroot = "/var/run/containers/storage"
+runroot = "/var/tmp/theia"
+
+# Primary Read/Write location of container storage
+# graphroot = "/var/lib/containers/storage"
```

Comment:
Is this one intentionally commented out?

Reply:
Yes, but has been removed.